tensorflow/tools/ci_build/ci_parameterized_build.sh

```diff
@@ -501,14 +501,21 @@ fi
 
 chmod +x ${TMP_SCRIPT}
 
+# Map TF_BUILD container types to containers we actually have.
+if [[ "${CTYPE}" == "android_nightly" ]]; then
```

Comment:
LGTM in general. However, can we call it "android_nightly" something more informative and less tightly coupled with our particular CI setup? My understanding is that "android_nightly" is longer than "android" and runs (a lot of?) tests? So can we call it something like "android_test" or "android_full" and rename the old one ("andrdoid") to "android_build_only"?